lib/imagery/iscatt_core.c

```diff
@@ -526,6 +526,7 @@ static int compute_scatts_from_chunk_row(struct scCats *scatt_conds,
 			      ("Unable to read from category raster condition file."));
 		    return -1;
 		}
+		/* following condition is never true, what is this supposed to test? */
 		if (n_pixs != n_pixs) {
```

Comment:
The `n_pix != n_pix` test is always false.

Reply:
n_pixs is the return value of the previous call to `fread()`, thus 
```
if (npixs != (row_size) / sizeof(unsigned char)) {
```
would make sense.

Reply:
Thanks! Added as suggested.